### PR TITLE
Log nr of socket events per wait

### PR DESCRIPTION
**Pull request by tmds**
cc @adamsitnik @antonfirsov

### CODE REVIEW

File: src/Tmds.LinuxAsync/SocketsEventSource.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using System.Diagnostics.Tracing;
+
+namespace Tmds.LinuxAsync
+{
+    [EventSource(Name = "SocketsEventSource")]
+    internal class SocketsEventSource : EventSource
+    {
+        [Event(1, Level = EventLevel.LogAlways)]
+        public void SocketEventsPerWait(int eventsCount) => WriteEvent(1, eventsCount);
```
Comment (tmds) [inline src/Tmds.LinuxAsync/SocketsEventSource.cs]: @adamsitnik I wonder about the added value of implementing this as an `EventCounter`. I guess the added value is mostly when you have some tooling that can work with `EventCounter`, but I don't think that applies for us. Does that make sense?

Comment (adamsitnik) [reply src/Tmds.LinuxAsync/SocketsEventSource.cs]: Honestly, I had no idea about the existence of [EventCounter](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md) but after reading the docs it looks like a great fit for this particular event. The question is - is it supported by Event Pipe? cc @noahfalk

Comment (tmds) [reply src/Tmds.LinuxAsync/SocketsEventSource.cs]: Yes, it's supported by EventPipe. `EventCounter` is the x-plat `PerformanceCounter`.
I'll turn this into an `EventCounter` then.

**Comment by tmds** [comment]
No longer working on this.

**Pull request closed without merging**